content/en/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md

```diff
@@ -119,7 +119,7 @@ The following manifest describes a single-instance MySQL Deployment. The MySQL c
 The `kustomization.yaml` contains all the resources for deploying a WordPress site and a 
 MySQL database. You can apply the directory by
 ```shell
-kubectl apply -k ./
+kubectl apply -f ./
```

Comment:
Hello @mudit000. To better understand the change, can you explain why the `-k flag` should be reverted? Are you using `kubectl v1.14`?